TabMonConfigBuilder/Helpers/StringHelpers.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+namespace TabMonConfigBuilder.Helpers
+{
+    /// <summary>
+    /// Static string helper class.
+    /// </summary>
+    class StringHelpers
+    {
+        public static readonly IList<string> InstructionSet = new List<string>()
+                    {
+                        "Usage Instructions!",
+                        "STEP 1: Verify that the host and process count are correct. If there",
+                        "        are any discrepancies, you may need to manually add or remove",
+                        "        hosts and processes.",
+                        "STEP 2: For each host, replace the worker in \"computerName\" and",
+                        "        \"address\" entry to reflect the correct host information.",
+                        "STEP 3: Copy the information below the line into the \"Cluster section\"",
+                        "        of the TabMon.Config.",
+                        "_______________________________________________________________________",
+                        ""
+                    };
+
+        public static readonly IDictionary<string, string> ProcessLookupDict = new Dictionary<string, string>
```

Comment:
What about other processes that are not in this list?

Reply:
We've limited to these processes for now. We can expand later.